azure/image/ubuntu/azure-ubuntu.pkr.hcl

```diff
@@ -9,7 +9,7 @@ source "azure-arm" "ubuntu" {
   image_publisher                   = "${var.publisher}"
   image_offer                       = "${var.offer}"
   image_sku                         = "${var.sku}"
-  managed_image_name                = "${var.az_image_name}"
+  managed_image_name                = "${var.az_image_name}-${var.az_gallery_image_version}"
```

Comment:
Why not override the `az_image_name` during start of the build? This way it leaves it to the user to decide how to name the image.

Reply:
Yes I agree. A user can choose to provide an image name with that format if they like. `PKR_VAR_az_image_name="$IMAGE_NAME-$PKR_VAR_az_gallery_image_version"`